README.md

```diff
@@ -141,64 +148,93 @@ If your organization uses a centralized log management product, see its document
 
  Keep remaining options with the default unless your network topology requires you to modify them.
 
-```
-  Usage: outsystemscc [options] <server> <remote> [remote] [remote] ...
+    Usage: outsystemscc [options] <server> <remote> [remote] [remote] ...
+
+    <server> is the URL to the server. Use the Address displayed on ODC Portal.
+
+    <remote>s are remote connections tunneled through the server, each of
+    which come in the form:
+
+        R:<local-port>:<remote-host>:<remote-port>
+
+    which does reverse port forwarding, sharing <remote-host>:<remote-port>
+    from the client to the server's <local-port>.
+
+        example remotes
+
+        R:8081:192.168.0.3:8393
+        R:8082:192.168.0.4:587
+
+        See https://github.com/OutSystems/cloud-connector for examples in context.
+        
+    Options:
+
+        --keepalive, An optional keepalive interval. Since the underlying
+        transport is HTTP, in many instances we'll be traversing through
+        proxies, often these proxies will close idle connections. You must
+        specify a time with a unit, for example '5s' or '2m'. Defaults
+        to '25s' (set to 0s to disable).
+
+        --max-retry-count, Maximum number of times to retry before exiting.
+        Defaults to unlimited.
+
+        --max-retry-interval, Maximum wait time before retrying after a
+        disconnection. Defaults to 5 minutes.
+
+        --proxy, An optional HTTP CONNECT or SOCKS5 proxy which will be
+        used to reach the server. Authentication can be specified
+        inside the URL.
+        For example, http://admin:password@my-server.com:8081
+                or: socks://admin:password@my-server.com:1080
+
+        --header, Set a custom header in the form "HeaderName: HeaderContent". 
+        Use the Token displayed on ODC Portal in using token as HeaderName.
+        
+        --hostname, Optionally set the 'Host' header (defaults to the host
+        found in the server url).
 
-  <server> is the URL to the server. Use the Address displayed on ODC Portal.
+        --pid Generate pid file in current working directory
 
-  <remote>s are remote connections tunneled through the server, each of
-  which come in the form:
+        -v, Enable verbose logging
 
-    R:<local-port>:<remote-host>:<remote-port>
+        --help, This help text
 
-  which does reverse port forwarding, sharing <remote-host>:<remote-port>
-  from the client to the server's <local-port>.
+    Signals:
+        The outsystemscc process is listening for:
+        a SIGUSR2 to print process stats, and
+        a SIGHUP to short-circuit the client reconnect timer
 
-    example remotes
+## 5. <a name="faqs"></a> FAQs <small><sup>[Top ▲](#table-of-contents)</sup></small>
 
-	  R:8081:192.168.0.3:8393
-    R:8082:192.168.0.4:587
+### How do I run `outsystemscc` on Azure Container Instances?
```

Comment:
As agreed, this section should not be here. We have a generic comment above, so a dedicated ACI section is not needed and should be removed.